app/static/js/slcManagement.js

```diff
@@ -30,7 +30,7 @@ function renew(){
       url: `/serviceLearning/renew/${courseID}/${termID}/`,
       type: "POST",
       success: function(s){
-        location.reload();
+        location = '/serviceLearning/editProposal/' +courseID;
```

Comment:
You are editing the wrong course. This is the edit page for the old course, not the new one.